core/state.go

```diff
@@ -497,7 +497,6 @@ func (s *State) updateDeclaredClassesTrie(declaredClasses map[felt.Felt]*felt.Fe
 			continue
 		}
 
-		// https://docs.starknet.io/documentation/starknet_versions/upcoming_versions/#commitment
```

Comment:
this one didn't make sense at all.

Reply:
Page has changed, here is what it [looked like before](https://web.archive.org/web/20230328171618/https://docs.starknet.io/documentation/starknet_versions/upcoming_versions/#commitment):

<img width="838" alt="Screenshot 2024-07-06 at 14 45 57" src="https://github.com/NethermindEth/juno/assets/775020/5da0ed7a-7773-4d70-acc9-3f54dd8cfc60">

I haven't found any relevant page currently